### DISCUSSION TITLE
Android client should reconnect to server after connection is lost

### DISCUSSION THREAD

**Discussion by Ramon-Salazar** [BearWare/TeamTalk5#2507] [Ideas]
### Description

Hi @bear101 

when disconnecting from a network to swich to another network the client should automaticly connect, but since 5.8 it doesn't happen and it's considered a bug

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [ ] Windows
- [ ] macOS
- [X] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

the client should reconnect as soon as when connected to a same network or a different network.

### Actual behavior

When disconnected from a network like wi-fi then swiching to a different network the clinet doesn't connect.

### Steps to reproduce problem

while connected swiching to a different wifi or different connection, like mobile data, sometimes the bug happens, or it happens when disconnected for few seconds.

**Comment by bear101** [comment]
When I tried to reproduce this issue it only occurs if TeamTalk is in background. Can you confirm this?
